### PR TITLE
release: On windows use .exe and alter README

### LINKED ISSUE #388: release: Use different README template for windows

### PR DESCRIPTION
**Pull request by nhooyr**
Closes #388

### CODE REVIEW

**Comment by nhooyr** [comment]
@alixander Let's release v0.1.1 with this fix?

File: ci/release/_build.sh [hunk 1]
```diff
@@ -4,19 +4,23 @@ cd -- "$(dirname "$0")/../.."
...
+sh_c cp ./ci/release/template/Makefile "$HW_BUILD_DIR"
+sh_c cp -R ./ci/release/template/man "$HW_BUILD_DIR"
+sh_c cp -Rf ./ci/release/template/scripts "$HW_BUILD_DIR"
+sh_c VERSION="$VERSION" ./ci/release/template/README.md.sh \> "'$HW_BUILD_DIR/README.md'"
 
 ensure_goos
 ensure_goarch
 sh_c mkdir -p "$HW_BUILD_DIR/bin"
-sh_c CGO_ENABLED=0 go build -trimpath \
+sh_c GOOS="$GOOS" GOARCH="$GOARCH" CGO_ENABLED=0 go build -trimpath \
```
Comment (alixander) [inline ci/release/_build.sh:16]: we don't need the cgo_enabled=0 anymore, unless i'm missing something

File: ci/release/changelogs/next.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 #### Improvements 🧹
 
+- The windows release binary is now suffixed correctly with `.exe`
```
Comment (alixander) [inline ci/release/changelogs/next.md:5]: Windows

File: ci/release/_build.sh (hunk 1 above)
Comment (nhooyr) [reply ci/release/_build.sh:16]: Go defaults to using libc for network resolution and os/user both of which I believe we import. `CGO_ENABLED=0` turns the libc integration off and uses pure Go implementations which should work in 99% of use cases and enable our binaries to be significantly more portable.

File: ci/release/template/README.md.sh [hunk 3]
```diff
@@ -1,12 +1,50 @@
...
+arch: $ARCH
+
+Built with $(go version | grep -o 'go[^ ]\+').
+EOF
+
+ensure_os
+if [ "$OS" = windows ]; then
+  cat <<EOF
+
+We currently do not have a \`.msi\` for automatic installation on Windows so this release
```
Comment (alixander) [inline ci/release/template/README.md.sh:22]: "an msi"

File: docs/INSTALL.md [hunk 4]
```diff
@@ -140,7 +140,7 @@ go install oss.terrastruct.com/d2@latest
 
 ### Source Release
 
-To install a proper release from source clone the repository and then:
```
Comment (alixander) [inline docs/INSTALL.md:143]: nice lol

**Review by alixander — APPROVED** [review]

**Pull request merged**